<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COrders</class>
 <widget class="QWidget" name="COrders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auftrags-Bearbeitung</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../MEGAMover/MEGAMover.qrc">
    <normaloff>:/icons/MM.png</normaloff>:/icons/MM.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="grpHead">
       <property name="title">
        <string>Kopf-Daten</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="4">
         <widget class="QComboBox" name="cmdUser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Auftrags-/Angebots-Nr.:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Bearbeiter:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txtAnr">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMask">
           <string>99/99/99/9999; </string>
          </property>
          <property name="maxLength">
           <number>13</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>__/__/__/____</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabAdress1">
        <attribute name="title">
         <string>Auftraggeber-Daten</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QGroupBox" name="grpAG">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Auftraggeber</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="AdressPad" name="padAG" native="true"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="pboxAG">
                  <item>
                   <widget class="QPushButton" name="cmdSrchAG">
                    <property name="text">
                     <string>Suchen</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="cmdNewAG">
                    <property name="text">
                     <string>Neu anlegen</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="grpRE">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Rechnungsempfänger</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="AdressPad" name="padRE" native="true"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="pboxRE">
                  <item>
                   <widget class="QPushButton" name="cmdSrchRE">
                    <property name="text">
                     <string>Suchen</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="cmdNewRE">
                    <property name="text">
                     <string>Neu anlegen</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QGroupBox" name="grpCommonData">
               <property name="title">
                <string>Allgemeine Angaben</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Status:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbStatus">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>Nur erfasst</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Angebot erstellt</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Auftrag bestätigt</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>In Abwicklung</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bereit zur Abrechnung</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Abgerechnet</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Erfasst am:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtCreated"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Bearbeitet: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtUpdated"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="grpOrderDial">
             <property name="title">
              <string>Auftragswahl</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="cmdNewAuf">
                <property name="text">
                 <string>Neuer Auftrag</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cmdPrevAuf">
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cmdSearchAuf">
                <property name="text">
                 <string>Suchen</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cmdNextAuf">
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cmdDeleteAuf">
                <property name="text">
                 <string>Löschen</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="cmdClose">
                <property name="text">
                 <string>Schließen</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabAdress2">
        <attribute name="title">
         <string>Leistungs-Orte</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabInvRows">
        <attribute name="title">
         <string>Leistungs-Daten</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabDates">
        <attribute name="title">
         <string>Termine</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdressPad</class>
   <extends>QWidget</extends>
   <header>adresspad.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../MEGAMover/MEGAMover.qrc"/>
 </resources>
 <connections/>
</ui>
